modules/nodes-nodes-working-setting-booleans.adoc

```diff
@@ -0,0 +1,56 @@
+[id="nodes-nodes-working-setting-booleans"]
+
+= Setting SELinux booleans
+
+{product-title} allows you to enable and disable a SELinux boolean on a RHEL CoreOC node. The following procedure explains how to modify SELinux booleans on nodes using the MachineConfigOperator (MCO). The `container_manage_cgroup` is the example sebool in this procedure. You can modify this value to whichever sebool you need.
+
+.Prerequisites:
```

Comment:
I don't think we need a colon after Prerequisites, as per what I'm seeing in the docs.